src/components/ValidatedInput/ValidFieldPassword.tsx

```diff
@@ -141,11 +157,36 @@ export class ValidFieldPassword extends React.Component<ValidFieldPasswordProps,
     );
   };
 
+  onChangeWrapper = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { onChange } = this.props;
+    if (onChange) onChange(event);
+    if (!event.target.value) return this.setState({ passwordStrenghtLevel: null });
+    const { score } = zxcvbn(event.target.value);
```

Comment:
return immediately if `hasPasswordStrenghtMeter` is false

Reply:
I was thinking to keep the strenght state updated so when `hasPasswordStrenghtMeter `set from false to true it will get the strenght and show it. but I change it so when the `hasPasswordStrenghtMeter ` change from false to true it will calculate password strength.

Thank you for catching this